include/rpt/parameters_rpt.h

```diff
@@ -0,0 +1,116 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 - 2021 by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+
+ *
+ * Author: Audrey Collard-Daigneault, Polytechnique Montreal, 2021-
+ */
+
+/*
+ *  This file defines parameters needed for the RPT simulation in the Parameters
+ * namespace.
+ */
+
+#ifndef lethe_rpt_parameters_h
+#define lethe_rpt_parameters_h
+
+#include <deal.II/base/config.h>
+
+#include <core/parameters.h>
+
+#include <deal.II/base/parameter_acceptor.h>
+#include <deal.II/base/parameter_handler.h>
+#include <deal.II/base/parsed_function.h>
+
+using namespace dealii;
+
+namespace Parameters
+{
+  /**
+   * @brief RPTParameters - Defines the common parameters for the RPT
+   * simulation as fixed parameters, number of Monte Carlo iterations and
+   * filename of the particle positions.
+   */
+
+  struct RPTParameters
+  {
+    // Particle positions filename
+    std::string particle_positions_file;
+
+    // Enable to export counts result in a .csv file
+    bool export_counts;
+
+    // Number of Monte Carlo iterations for alpha and theta
+    unsigned int iteration_number;
```

Comment:
```suggestion
    unsigned int n_monte_carlo_iterations
```
Use a variable name that is a bit more explicit :)